core/controllers/reader.py

```diff
@@ -240,121 +240,63 @@ class StorePlaythroughHandler(base.BaseHandler):
     the playthrough already exists, it is updated in the datastore.
     """
 
-    def __init__(self, *args, **kwargs):
-        """Method that initializes member variables for the handler.
-
-        Attributes:
-            current_exp_issues: ExplorationIssues. The exploration issues domain
-                object.
-            current_issue_schema_version: int. The issue schema version.
-            current_playthrough_id: str|None. The Playthrough ID or None.
-        """
-        super(StorePlaythroughHandler, self).__init__(*args, **kwargs)
-        self.current_exp_issues = None
-        self.current_issue_schema_version = None
-        self.current_playthrough_id = None
-
-    def _find_matching_issue_in_exp_issues(self, playthrough):
+    def _get_matching_exp_issue(self, playthrough, exp_issues):
         """Finds an issue with the equivalent issue_type and associated states
         as the given playthrough in the unresolved issues list of the
         exploration issues model.
 
         Args:
             playthrough: Playthrough. The playthrough domain object.
+            exp_issues: ExplorationIssues. The exploration issues domain object.
 
         Returns:
-            int|None. The index at which the issue was found, None otherwise.
+            ExplorationIssue or None.
```

Comment:
Missing description. (Maybe worth rewording the top-level docstring above to be clearer grammatically/semantically, too -- can't tell whether the issue in question is supposed to be unique, or one of many.)

Reply:
Done.